doc/man1/openssl-genpkey.pod.in

```diff
@@ -72,10 +72,13 @@ Do not output "status dots" while generating keys.
 The output file password source. For more information about the format of I<arg>
 see L<openssl-passphrase-options(1)>.
 
-=item B<-I<cipher>>
+=item B<-cipher> I<alg>
 
-This option encrypts the private key with the supplied cipher. Any algorithm
-name accepted by EVP_get_cipherbyname() is acceptable such as B<des3>.
+Encrypts the private key using the specified algorithm. The algorithm can be
+specified using a name that is accepted by the EVP_get_cipherbyname() function.
+For example, use the syntax B<-aes-128-cbc> to specify the AES encryption
+algorithm with a 128-bit key in CBC mode. Other supported algorithms include
+B<-des3> for 3DES encryption.
```

Comment:
Suggest ditching the final sentence.  3DES is old and weak and shouldn't be encouraged.